File: chatwoot/typescript-node/src/AddNewAgentToTeamExample.ts
import * as fs from 'fs';
import api from "chatwoot_client"
import models from "chatwoot_client"

const apiCaller = new api.TeamsApi();
apiCaller.setApiKey(api.TeamsApiApiKeys.userApiKey, "USER_API_KEY");

const addNewAgentToTeamRequest: models.AddNewAgentToTeamRequest = {
  userIds: [
  ],
};

apiCaller.addNewAgentToTeam(
  0, // accountId
  0, // teamId
  addNewAgentToTeamRequest, // data
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling TeamsApi#addNewAgentToTeam:");
  console.log(error.body);
});

File: chatwoot/kotlin/src/main/kotlin/oseg/chatwoot_examples/AddNewAgentToTeamExample.kt
package oseg.chatwoot_examples

import com.chatwoot.client.infrastructure.*
import com.chatwoot.client.apis.*
import com.chatwoot.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class AddNewAgentToTeamExample
{
    fun addNewAgentToTeam()
    {
        ApiClient.apiKey["api_access_token"] = "USER_API_KEY"

        val addNewAgentToTeamRequest = AddNewAgentToTeamRequest(
            userIds = listOf (),
        )

        try
        {
            val response = TeamsApi().addNewAgentToTeam(
                accountId = 0,
                teamId = 0,
                _data = addNewAgentToTeamRequest,
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling TeamsApi#addNewAgentToTeam")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling TeamsApi#addNewAgentToTeam")
            e.printStackTrace()
        }
    }
}
